src/main/scripts/org/reaktivity/specification/nukleus/amqp/streams/send.to.client.through.multiple.sessions/client.rpt

```diff
@@ -0,0 +1,80 @@
+#
+# Copyright 2016-2020 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+connect await ROUTED_CLIENT
```

Comment:
We need two different streams here to model two different links.

Suggest writing the script as if there is only one link, then repeat the entire `connect` stream, adjusting the `channel` to `1`.  The companion `accept` script will need only one `accept` but two `accepted`.